ext/sqlite3/tests/bug68760.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Bug #68760 (Callback throws exception behaviour.)
```

Comment:
@cjbj it would be better to enclose code in your comment in [code block](https://gist.github.com/Kubo2/42b33cb7e9f15584b705) to prevent intepretation of Markdown.